### ISSUE TITLE
Polyglot no longer obscuring text for players after 0.7.10 upgrade

### ISSUE THREAD

**Issue by phil-the-dev** [League-of-Foundry-Developers/fvtt-module-polyglot#143]
**Describe the bug**
I believe it began when I upgraded to version 0.7.10, but as the GM I can select a different language and it works perfectly, but my players no longer are able to do the same. It is just sent as regular text as if Common was selected.

**To Reproduce**
1. Log in as a Player
2. Select a Language that is not Common
3. Type a message
4. Hit 'Enter'
5. Text is posted normally, without any obfuscation.

**Expected behavior**
A message has obscured text, and a translation subsection for those who know the language.

**Screenshots**
![image](https://user-images.githubusercontent.com/1228950/121098365-b4c08c80-c7c3-11eb-908e-d7a2a97ee271.png)

 - Game System and version: Foundry 0.7.10, DnD 5e: 1.2.4
 - Polyglot Version: 1.5.18.2
 - Are you using Babele? No

**Additional context**
This was working before, had a whole session where my players were fooling around with it. I've tried disabling it and re-enabling it, as well as uninstalling/reinstalling.

**Comment by mclemente** [comment]
Does Kaileigh have a character on the map? If she doesn't, you need to change the `Scramble OOC Chat Messages` setting, otherwise it won't change their language.

**Comment by phil-the-dev** [comment]
ohh, ok, so it does scramble when her token is on the map. I have the Scramble on OOC Chat messages selected, but that seems to be a GM only feature. I guess nothing is broken here, I didn't realize you needed a token on the map. I would like the ability for players to still chat in different language regardless if their token is out on the map, but that seems like a new feature request entirely.

Thanks for being so timely on this issue!

**Comment by mclemente** [comment]
TL;DR: It won't be an issue whenever you update to 0.8.

Oh, I forgot it was a GM-only setting on 0.7, that's why I said to change the setting, sorry about that.
The good news is that someone else ran into this problem before (https://github.com/League-of-Foundry-Developers/fvtt-module-polyglot/issues/133) and I made it available to players.
The bad news is that it's 0.8-exclusive because I was already working on the 0.8 update back then and handling two versions of Foundry is a bit of a hassle.
